frontend/scss/components/molecules/pixi-primary-metric.scss

```diff
@@ -99,11 +99,10 @@
         margin-right: 20px;
       }
 
-      span,
+      a,
```

Comment:
￼this is a super generic selector, are you sure there are no side effects? Would it be possible to add a class rather than using a tag?

Reply:
Yeah I can do that.

Reply:
Actually, this one is only on `a` descendants of the `ap-m-pixi-primary-metric-body-result` class, so should be safe.

Reply:
ah, I missed this was a descendant selector-thanks!